ape_trezor/exceptions.py

```diff
@@ -0,0 +1,33 @@
+from ape.exceptions import AccountsError
+
+
+class TrezorAccountException(AccountsError):
+    """
+    An error that occurs in the ape Trezor plugin.
+    """
+
+
+class TrezorSigningError(TrezorAccountException):
+    """
+    An error that occurs when signing a message or transaction
+    using the Trezor plugin.
+    """
+
+
+class TrezorUsbError(TrezorAccountException):
```

Comment:
Maybe we should call this one Client instead of Usb because so it does not seem like we are implying that we are doing the low-level USB stuff (as we have to do in the ledger plugin)

```suggestion
class TrezorClientError(TrezorAccountException):
```